lib/ReplacePointerBitcastPass.cpp

```diff
@@ -925,7 +925,7 @@ bool ReplacePointerBitcastPass::runOnModule(Module &M) {
           }
         } else {
 
-          errs() << "About to crash at 820" << M << "Aboot\n\n";
+          errs() << "About to crash at 820" << M << "Abort\n\n";
```

Comment:
You can just delete this line.

Reply:
My bad.  I inserted it a long time ago so I could track down bad behaviour.  I never intended to check it in.  :-)